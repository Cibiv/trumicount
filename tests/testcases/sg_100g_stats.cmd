$TRUMICOUNT --input-umitools-group-out "$TESTCASES/sg_100g.gout.bz2" \
	--paired --filter-strand-umis --umipair-sep '-' \
	--molecules 1 --threshold 24 \
	--include-filter-statistics \
	--output-counts sg_100g_stats.output --digits 2
