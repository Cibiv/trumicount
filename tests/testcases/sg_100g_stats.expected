gene	n.umis	n.tot	efficiency	depth	loss	n.obs	n.reads	n.umis.prefilter	n.reads.prefilter
AAC73112	10	10	0.82	72	0.026	10	739	10	739
AAC73113	34	35	0.82	71	0.026	34	2442	38	2498
AAC73114	14	14	0.82	71	0.026	14	949	16	986
AAC73115	18	18	0.82	71	0.026	18	1222	22	1259
AAC73117	2	2.1	0.82	72	0.026	2	145	2	145
AAC73119	154	158	0.82	74	0.026	154	11684	176	11904
AAC73120	11	11	0.82	72	0.026	11	792	12	814
AAC73121	2	2.1	0.82	72	0.026	2	223	2	223
AAC73125	907	931	0.82	74	0.026	907	68788	1006	69715
AAC73126	123	126	0.82	70	0.026	123	8593	136	8724
AAC73130	7	7.2	0.82	72	0.026	7	533	8	535
AAC73134	61	63	0.82	77	0.026	61	5024	66	5056
AAC73136	33	34	0.82	70	0.026	33	2265	44	2403
AAC73137	185	190	0.82	69	0.026	185	12928	214	13163
AAC73138	12	12	0.82	72	0.026	12	827	14	849
AAC73139	32	33	0.82	68	0.026	32	2008	34	2049
AAC73140	17	17	0.82	71	0.026	17	1168	20	1216
AAC73141	10	10	0.82	72	0.026	10	729	12	765
AAC73142	4	4.1	0.82	72	0.026	4	288	4	288
AAC73143	59	61	0.82	71	0.026	59	4267	66	4314
AAC73144	115	118	0.82	70	0.026	115	8103	138	8278
AAC73150	2	2.1	0.82	72	0.026	2	74	2	74
AAC73151	2	2.1	0.82	72	0.026	2	162	2	162
AAC73157	2	2.1	0.82	72	0.026	2	144	2	144
AAC73158	7	7.2	0.82	70	0.026	7	411	8	412
AAC73159	4	4.1	0.82	72	0.026	4	295	4	295
AAC73160	6	6.2	0.82	73	0.026	6	482	6	482
AAC73161	4	4.1	0.82	72	0.026	4	311	4	311
AAC73162	30	31	0.82	73	0.026	30	2316	32	2321
AAC73163	30	31	0.82	72	0.026	30	2171	34	2191
AAC73164	66	68	0.82	71	0.026	66	4662	72	4722
AAC73165	61	63	0.82	76	0.026	61	4955	66	4983
AAC73166	2	2.1	0.82	72	0.026	2	159	2	159
AAC73169	1	1	0.82	72	0.026	1	29	2	35
AAC73170	32	33	0.82	72	0.026	32	2335	40	2451
AAC73171	6	6.2	0.82	73	0.026	6	520	8	564
AAC73174	4	4.1	0.82	72	0.026	4	210	4	210
AAC73175	4	4.1	0.82	72	0.026	4	208	4	208
AAC73176	2	2.1	0.82	72	0.026	2	175	2	175
AAC73178	5	5.1	0.82	71	0.026	5	287	6	297
AAC73179	4	4.1	0.82	72	0.026	4	332	4	332
AAC73180	7	7.2	0.82	70	0.026	7	413	8	433
AAC73182	2	2.1	0.82	72	0.026	2	131	2	131
AAC73183	15	15	0.82	72	0.026	15	1126	16	1131
AAC73184	6	6.2	0.82	72	0.026	6	463	6	463
AAC73185	4	4.1	0.82	72	0.026	4	338	4	338
AAC73186	2	2.1	0.82	72	0.026	2	137	2	137
AAC73189	4	4.1	0.82	72	0.026	4	313	4	313
AAC73191	22	23	0.82	73	0.026	22	1673	24	1683
AAC73192	17	17	0.82	71	0.026	17	1176	20	1219
AAC73193	18	18	0.82	71	0.026	18	1208	18	1208
AAC73194	6	6.2	0.82	72	0.026	6	436	6	436
AAC73195	28	29	0.82	74	0.026	28	2254	34	2345
AAC73196	28	29	0.82	73	0.026	28	2169	32	2200
AAC73197	42	43	0.82	71	0.026	42	2949	50	3041
AAC73198	15	15	0.82	72	0.026	15	1147	18	1164
AAC73199	20	21	0.82	69	0.026	20	1228	22	1237
AAC73200	5	5.1	0.82	70	0.026	5	294	6	297
AAC73201	11	11	0.82	74	0.026	11	928	12	944
AAC73202	19	20	0.82	71	0.026	19	1289	22	1304
AAC73203	13	13	0.82	72	0.026	13	999	16	1024
AAC73204	23	24	0.82	72	0.026	23	1672	24	1684
AAC73205	16	16	0.82	70	0.026	16	1018	18	1062
AAC73206	27	28	0.82	70	0.026	27	1809	32	1873
AAC73207	88	90	0.82	72	0.026	88	6471	102	6589
AAC73208	24	25	0.82	68	0.026	24	1432	26	1461
AAC73209	99	102	0.82	72	0.026	99	7267	114	7417
AAC73210	2	2.1	0.82	72	0.026	2	108	2	108
AAC73212	4	4.1	0.82	72	0.026	4	266	4	266
AAC73213	6	6.2	0.82	73	0.026	6	498	6	498
AAC73214	8	8.2	0.82	72	0.026	8	601	8	601
ACO59989	23	24	0.82	75	0.026	23	1889	24	1903
